core/src/App.html

```diff
@@ -545,9 +547,37 @@
         });
       },
       onResize() {
-        if (window.innerWidth >= CSS_BREAKPOINTS.desktopMinWidth) {
+        const simpleDesktopToMobile = () => {
+          if (this.get().responsiveNavSetting == 'simple') {
+            if (
+              window.innerWidth < CSS_BREAKPOINTS.desktopMinWidth &&
+              this.get().previousBreakpoint >= CSS_BREAKPOINTS.desktopMinWidth
+            ) {
+              this.closeLeftNav();
```

Comment:
Correct me if I am wrong, but `closeLeftNav` is closing the _mobile_ left nav. In that case, we could rename the method to sth. like `closeMobileLeftNav` ;)

Reply:
As discussed, it's not only for mobile navigation but as well for desktop in `simple` mode.